test/datasets/test_data_frame_text_benchmark.py

```diff
@@ -0,0 +1,105 @@
+import tempfile
+
+import pytest
+
+from torch_frame.config.text_embedder import TextEmbedderConfig
+from torch_frame.datasets import DataFrameTextBenchmark
+from torch_frame.testing.text_embedder import HashTextEmbedder
+from torch_frame.typing import TaskType
+
+
+@pytest.mark.parametrize('scale', ["small", "medium", "large"])
+@pytest.mark.parametrize('task_type', [
+    TaskType.BINARY_CLASSIFICATION, TaskType.MULTICLASS_CLASSIFICATION,
+    TaskType.REGRESSION
+])
+def test_data_frame_text_benchmark_match(task_type, scale):
+    # Make sure task_type, scale, idx triple map to the fixed underlying
+    # dataset. New dataset can be appended, but the existing mapping needs to
+    # be preserved.
+    datasets = DataFrameTextBenchmark.datasets_available(
+        task_type=task_type, scale=scale)
+    if task_type == TaskType.BINARY_CLASSIFICATION:
+        if scale == 'small':
+            assert datasets[0] == ('MultimodalTextBenchmark', {
+                'name': 'fake_job_postings2'
+            })
+        elif scale == 'medium':
+            assert datasets[0] == ('MultimodalTextBenchmark', {
+                'name': 'jigsaw_unintended_bias100K'
+            })
+        elif scale == 'large':
+            assert len(datasets) == 0
+    elif task_type == TaskType.MULTICLASS_CLASSIFICATION:
+        if scale == 'small':
+            assert datasets[0] == ('MultimodalTextBenchmark', {
+                'name': 'product_sentiment_machine_hack'
+            })
+            assert datasets[1] == ('MultimodalTextBenchmark', {
+                'name': 'news_channel'
+            })
+        elif scale == 'medium':
+            assert datasets[0] == ('MultimodalTextBenchmark', {
+                'name': 'wine_reviews'
+            })
+            assert datasets[1] == ('HuggingFaceDatasetDict', {
+                'path':
+                'maharshipandya/spotify-tracks-dataset',
+                'columns': [
+                    'artists', 'album_name', 'track_name', 'popularity',
+                    'duration_ms', 'explicit', 'danceability', 'energy', 'key',
+                    'loudness', 'mode', 'speechiness', 'acousticness',
+                    'instrumentalness', 'liveness', 'valence', 'tempo',
+                    'time_signature', 'track_genre'
+                ],
+                'target_col':
+                'track_genre',
+            })
```

Comment:
Is this the only one from `HuggingFace`? Can we add a few more?

Reply:
I will look at for follow up